src/index.js

```diff
@@ -59,6 +61,7 @@ class CgSlider extends EventEmitter {
     ariaLabel: '',
     ariaLabelledBy: '',
     ariaDescribedBy: '',
+    ariaValuetext: val => +val
```

Comment:
According to the docs this function must return `string`. Current default implementation returns `number`.